src/app/modules/main/wallet_section/wallet_connect/controller.nim

```diff
@@ -37,6 +38,8 @@ QtObject:
   proc finishSessionRequest(self: Controller, signature: string)
   proc finishAuthRequest(self: Controller, signature: string)
 
+  proc requestOpenPairModal*(self: Controller, uri: string) {.signal.}
```

Comment:
Maybe to rename it to a more general name, like `requestOpenWalletConnectPopup`?